crates/biome_js_analyze/tests/specs/nursery/useQwikValidLexicalScope/invalid.jsx

```diff
@@ -0,0 +1,21 @@
+import { component$, useTask$, $ } from '@builder.io/qwik';
```

Comment:
```suggestion
// should emit diagnostics
import { component$, useTask$, $ } from '@builder.io/qwik';
```

And that's why you missed the absence of the diagnostics. You should review the snapsthos just in case, always